### PR TITLE
Initial page of CS8802 error

### LINKED ISSUE #38811: error CS8802: Only one compilation unit can have top-level statements.
### Type of issue

Missing information

### Description

[Enter feedback here]
"error CS8802: Only one compilation unit can have top-level statements." is displaying in my device please fix it

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error?f1url=%3FappId%3Droslyn%26k%3Dk(CS8802)

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md

### Document Version Independent Id

58a19505-ac46-c5c8-12cc-fe87b6aa18a7

### Article author

@BillWagner

### Metadata

* ID: bbb051ee-f020-66ba-c601-4a08ad32ef61 
* Product: **dotnet-csharp**
* Technology: **csharp-diagnostics**

### PR DESCRIPTION
**Pull request by BartoszKlonowski**
This pull request closes #38811 

It introduces the page describing CS8802 error along with a simple example and explanation.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/compiler-messages/cs8802.md](https://github.com/dotnet/docs/blob/9d7cb83b10b471e20a160acd45efc5468a98d59e/docs/csharp/language-reference/compiler-messages/cs8802.md) | [Compiler Error CS8802](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs8802?branch=pr-en-us-43728) |
| [docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md](https://github.com/dotnet/docs/blob/9d7cb83b10b471e20a160acd45efc5468a98d59e/docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md) | [Sorry, we don't have specifics on this C# error](https://review.learn.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error?branch=pr-en-us-43728) |

### CODE REVIEW

File: docs/csharp/language-reference/compiler-messages/cs8802.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+f1_keywords:
+  - "CS8802"
+helpviewer_keywords:
+  - "CS8802"
+---
+# Compiler Error CS8802
+
+Only one compilation unit can have top-level statements.
+
+This error indicates that there are two or more [top-level statements](../../fundamentals/program-structure/top-level-statements.md) in a single compilation unit (single solution, project or a single group of files compiled into a single binary file).
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-messages/cs8802.md:14]: I don't think "single solution" is right.

File: docs/csharp/language-reference/compiler-messages/cs8802.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+description: "Compiler Error CS8802"
+title: "Compiler Error CS8802"
+ms.date: 7/01/2024
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-messages/cs8802.md:4]:
```suggestion
ms.date: 11/24/2024
```

File: docs/csharp/language-reference/compiler-messages/cs8802.md (hunk 1 above)
Comment (BillWagner) [reply docs/csharp/language-reference/compiler-messages/cs8802.md:14]: I agree. A solution can contain multiple console applications:

```suggestion
This error indicates that there are two or more [top-level statements](../../fundamentals/program-structure/top-level-statements.md) in a single compilation unit (single project or a single group of files compiled into a single binary file).
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks again @BartoszKlonowski 

In addition to the changes noted in this file, can you remove `CS8802` from the "sorry we don't have information on this error" file here: https://github.com/dotnet/docs/blob/main/docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md?plain=1#L428

**Comment by BartoszKlonowski** [comment]
Ah, indeed, sorry @BillWagner, I just keep on forgetting to remove those :(

**Review by BillWagner — APPROVED** [review]

**Pull request merged**